Example/Vouched/IdViewController.swift

```diff
@@ -26,18 +31,18 @@ class IdViewController: UIViewController {
     @IBOutlet private weak var confirmIdButton: UIButton!
     @IBOutlet private weak var retryIdButton: UIButton!
     @IBOutlet private weak var confirmPanel: UIView!
-
+    
     var inputFirstName: String = ""
     var inputLastName: String = ""
     var useCameraFlash = false
     var confirmID = false
     // temp storage of current ID to be confirmed
     var confirmIDDetectionResult: CardDetectResult?
-
+    
     private var helper: VouchedCameraHelper?
     private var detectionMgr: VouchedDetectionManager?
-    private let session = VouchedSession(apiKey: getValue(key:"API_KEY"), sessionParameters: VouchedSessionParameters())
-
+    private let session = VouchedSession(apiKey: getValue(key:"API_KEY"), sessionParameters: VouchedSessionParameters(), apiUrl: getValue(key:"API_URL"))
```

Comment:
This is more of a question than an action item.  How are we handling the case when the `API_URL` or `API_KEY` are missing?